### PR TITLE
Updated readme.md file

### PR DESCRIPTION
**Pull request by goodnesskay**
I have updated the readme.md file. I worked on brief detail about the service,prerequisites,getting started, installation,built with,contributing ,license and authors. Please check the update and give feedback.

### CODE REVIEW

**Comment by perfectmak** [comment]
@goodnesskay Kindly Resolve the Conflict first and update the PR before I take a look

**Comment by goodnesskay** [comment]
I have resolved it @perfectmak Got what you asked me to look out

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
 # voucher-service
```
Comment (perfectmak) [inline README.md:1]: Could you change this 'voucher-service' to 'A Voucher Management as a Service Application'

**Comment by perfectmak** [comment]
Alright, this looks super cool and okay, just make the final edit in my comment and I'll merge this.

File: README.md (hunk 1 above)
Comment (goodnesskay) [reply README.md:1]: ok. I will now

Comment (goodnesskay) [reply README.md:1]: I think I have done it

Comment (perfectmak) [reply README.md:1]: Your change is not reflecting here. Ensure that you have pushed your changes up

**Pull request merged**